docs/3.0rc/deploy/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Deploy flows
-description: Learn how to configure flows for scheduled and remote execution.
+title: Deployment overview
+description: Learn what deployments are and how to use them in Prefect.
 tags:
```

Comment:
```suggestion
```

Reply:
tags and search->boost are left over metadata from mkdocs that we should remove.